### PR TITLE
Subtraction Operation

### PR DESCRIPTION
**Pull request by rmzi**
In this PR, I attempt to implement subtraction for our simple calculator.

I defined all business logic in the FE code itself, and my BE is a simple Python server running via `python -m SimpleHTTPServer`.

What do you think?

### CODE REVIEW

**Comment by rmzi** [comment]
@wilhelmwillie HELP! LOOK AT MY CODE PLEASE!!!!! AHH

👍

**Comment by WilhelmWillie** [comment]
yo your code is kinda whack man 
img_0235

File: index.html [hunk 1]
```diff
@@ -21,6 +21,7 @@
       var val2 = parseInt(document.getElementById("val2").value);
       document.getElementById("result").value = val1 + val2;
     }
+
```
Comment (WilhelmWillie) [inline index.html:24]: Where's the sub function??:?:!?!?!?!?111

**Review by WilhelmWillie — CHANGES_REQUESTED** [review]
Add sub function

File: index.html (hunk 1 above)
Comment (rmzi) [reply index.html:24]: OHHHH SNAP! I'll do that now.

**Comment by rmzi** [comment]
@WilhelmWillie Do you think this works right now??!?!?!?!?!?!??211/1/11?!

**Comment by WilhelmWillie** [comment]
LGTM 
download

**Comment by rmzi** [comment]
Fixed Bug #4

**Pull request merged**